comp/logs/agent/agent.go

```diff
@@ -131,6 +131,7 @@ func newLogsAgent(deps dependencies) provides {
 	return provides{
 		Comp:           optional.NewNoneOption[Component](),
 		StatusProvider: statusComponent.NewInformationProvider(NewStatusProvider()),
+		FlareProvider:  flaretypes.NewProvider(func(flaretypes.FlareBuilder) error { return nil }),
```

Comment:
Could you add a `NewNoneProvider`/`NewEmptyProvider`  helper to the flare comp for this (Or anything with a better name) ?

Reply:
Done here fa3b6bc75c8280b1b08d31b2bca9a1040f484483